scripts/drag_scripts/drag_helper2.py

```diff
@@ -0,0 +1,58 @@
+import pyautogui
+
+pyautogui.PAUSE = .05
+print (pyautogui.position())
+
+positions = []
+
+
+print("input top sections")
+input("1")
+
```

Comment:
Minor: there's a lot of vertical whitespace in this script.  You can compress it a bit.  Related code can be contiguous, separate code can be separated by one blank line.  I'd nestle the `input` up against the `position()` call it applies to, and separate each input group from the other by one blank line.